assets/app/scripts/directives/nav.js

```diff
@@ -58,10 +58,11 @@ angular.module('openshiftConsole')
 
           sortedProjects = $filter('orderByDisplayName')(projects);
           options = _.map(sortedProjects, function(item) {
+            // truncates @ 35 as this prevents words from clobbering the caret
             return $('<option>')
-                      .attr("value", item.metadata.name)
+                      .attr('value', $filter('truncate')(item.metadata.name, 35))
```

Comment:
That snippet of CSS still exists & the selector is still correct, but the `.truncate()` mixin no longer works.  It has a note:

```css
.truncate {
  // Bootstrap mixin
  // Note: If truncating within a flex container then the parent needs a width
  .text-overflow();
}
```
(@sg00dwin for reference)